src/Route.elm

```diff
@@ -79,41 +82,64 @@ fromUrl url =
 
 
 routeToString : Route -> String
-routeToString page =
-    let
-        ( path, query ) =
-            routeToPieces page
-    in
-    "/" ++ String.join "/" path ++ "?" ++ String.join "&" (List.filterMap Basics.identity query)
+routeToString =
+    uncurry Builder.absolute << routeToPieces
 
 
-routeToPieces : Route -> ( List String, List (Maybe String) )
+{-| Fixes issue with elm/url not properly escaping string
+-}
+builderString : String -> String -> QueryParameter
+builderString name =
+    Builder.string name << Url.percentEncode
```

Comment:
Since [Domen's PRs](https://github.com/elm/url/pull/34) are still not merged we need to define fixed version ourselves.

Reply:
:100: